packages/breadboard/package.json

```diff
@@ -61,6 +65,12 @@
       ]
     },
     "test": {
+      "dependencies": [
+        "test:ava",
+        "test:node"
+      ]
+    },
+    "test:ava": {
```

Comment:
I do support the ava -> node test conversion, but I think it would be better to do a package-by-package conversion of the existing tests, rather than adding new tests alongside the ava ones. Otherwise we're just adding complexity with more layers of tooling.

Reply:
I shall have a crack at converting the main breadboard tests. However, if I hit any major speed bumps I may start with one of the smaller packages

Reply:
On second thought, it will be nice to have a template for node tests checked in that we can iterate from, so actually I'd support landing this PR.

I'm a bit confused about how this PR relates to https://github.com/breadboard-ai/breadboard/pull/587 and https://github.com/breadboard-ai/breadboard/pull/588 though. Which ones need to land first, or can some of them be closed?